src/uharfbuzz/charfbuzz.pxd

```diff
@@ -197,3 +203,6 @@ cdef extern from "hb-ot.h":
         unsigned int start_offset,
         unsigned int* script_count,  # in/out
         hb_tag_t* script_tags)  # out
+
+    # hb-ot-font.h
+    void hb_ot_font_set_funcs(hb_font_t *font)
```

Comment:
nit: so far I put the stars before the space

Reply:
I copy-pasted it from the harfbuzz headers, which put the stars after the space, but I don't mind.